### PR TITLE
 informed that the Zoonotify Webapp has not been optimized for mobile screens

### PR DESCRIPTION
**Pull request by mahtabiltarabian**
…

### CODE REVIEW

File: src/app/App.tsx [hunk 1]
```diff
@@ -8,11 +8,29 @@ import { znTheme } from "./shared/style/Style-MainTheme";
...
+    const userAgent = navigator.userAgent.toLowerCase();
+    const isMobile = /iphone|ipod|android|ie|blackberry|fennec/.test(userAgent);
+    return isMobile;
+}
+
+function App(): ReactElement {
+    useEffect(() => {
+        if (detectMobileDevice()) {
+            alert(
+                "Please note: The Zoonotify Webapp has not been optimized for mobile screens."
```
Comment (dptolle) [inline src/app/App.tsx:21]: This should be a TOKEN with German and English translation provided by the language files

**Review by dptolle — CHANGES_REQUESTED** [review]

File: src/app/App.tsx (hunk 1 above)
Comment (mahtabiltarabian) [reply src/app/App.tsx]: ok

File: src/app/App.tsx [hunk 2]
```diff
@@ -1,18 +1,41 @@
...
+}
+
+function App(): ReactElement {
+    const { t } = useTranslation(["ErrorPage"]);
+
+    useEffect(() => {
+        if (detectMobileDevice()) {
+            alert(
+                t(
+                    "PLEASE NOTE: THE ZOONOTIFY WEBAPP HAS NOT BEEN OPTIMIZED FOR MOBILE SCREENS."
```
Comment (dptolle) [inline src/app/App.tsx:25]: Please use small one word token

File: src/app/App.tsx [hunk 3]
```diff
@@ -1,18 +1,41 @@
...
+    const userAgent = navigator.userAgent.toLowerCase();
+    const isMobile = /iphone|ipod|android|ie|blackberry|fennec/.test(userAgent);
+    return isMobile;
+}
+
+function App(): ReactElement {
+    const { t } = useTranslation(["ErrorPage"]);
+
+    useEffect(() => {
+        if (detectMobileDevice()) {
```
Comment (dptolle) [inline src/app/App.tsx:22]: PLease use existing Warning system

**Review by dptolle — CHANGES_REQUESTED** [review]

**Pull request closed without merging**